src/v/raft/follower_stats.h

```diff
@@ -24,6 +24,7 @@ class follower_stats {
     using container_t = absl::node_hash_map<vnode, follower_index_metadata>;
     using iterator = container_t::iterator;
     using const_iterator = container_t::const_iterator;
+    using value_type = container_t::value_type;
```

Comment:
why not just use `auto &` in the lambda?

Reply:
it is creating a template specialization, specifying a type explicitly makes compiling faster

Reply:
i wonder if it matters much in `.cc` files to worry about that compilation speed.

one concrete benefit to avoiding `auto` in this case, though, is that clangd doesn't understand what the type is. at least not int he verison i'm using.